docs/ray.md

```diff
@@ -78,6 +78,39 @@ used by the Ray workers, so even native dependencies (ex: cuda) are available.
 - Execute your training code on the Ray cluster.
 - Tear down the cluster when training terminates (successfully or with a failure).
 
+### Architecture
+
+![Ray/Sematic architecture diagram](./images/rayDiagram.png)
+
+When you use `with RayCluster(...):` inside a Sematic func, your Sematic func
+asks the Sematic server to allocate a Ray cluster. The Sematic server generates
+a manifest describing that cluster, which it then posts for
+[KubeRay](https://github.com/ray-project/kuberay) to use. KubeRay initializes
```

Comment:
Please mention that KubeRay was already installed and is running as a separate service in the cluster.

Reply:
that's mentioned in the setup instructions at the bottom of this docs page